app/recordtransfer/templates/recordtransfer/system_error.html

```diff
@@ -1,13 +1,28 @@
-{% extends 'recordtransfer/base.html' %}
+{% extends "recordtransfer/base.html" %}
 {% load i18n %}
-
-{% block title %}{% trans "System Error" %}{% endblock %}
-
+{% load static %}
+{% block title %}
+    {% trans "System Error" %}
+{% endblock title %}
 {% block content %}
-{% blocktrans %}
-<div>
-    <p>We're sorry, but this site is experiencing some issues.</p>
-    <p>An administrator has been notified. Please try back later.</p>
-</div>
-{% endblocktrans %}
-{% endblock %}
+    <div class="flex flex-col items-center justify-center min-h-[30vh] text-center py-10 px-4">
+        <!-- Error Message -->
+        <div class="max-w-md">
+            <div class="text-2xl font-bold text-primary px-4 py-2 ">{% trans "System Error" %}</div>
+            <p class="mb-4 text-gray-700 text-lg font-medium leading-relaxed">
+                {% trans "We're sorry, but this site is experiencing some issues." %}
+            </p>
+            <p class="text-gray-600 text-base leading-relaxed">
+                {% trans "An administrator has been notified. Please try back later." %}
```

Comment:
In many cases, a 500 error *won't* send an email, so I'd be more comfortable changing this text to:

```suggestion
                {% trans "Please try again later. Contact us if you continue to experience issues." %}
```